Release 4.15.0 (#194)
blackbaud/skyux-lookup: CHANGELOG.md

# 4.15.0 (2020-06-15)

> - Updated the lookup component to include a button which triggers the show more picker from the input box. [#183](https://github.com/blackbaud/skyux-lookup/pull/183)

Reviewer: ```suggestion
- Updated the lookup component to include a search button in the input field that opens the show more picker. [#183](https://github.com/blackbaud/skyux-lookup/pull/183)
```
"search button" and "input field" are the terminology in the design guidelines. If we actually mean the

> - Updated the lookup component's dropdown keyboard interactions. [#176](https://github.com/blackbaud/skyux-lookup/pull/176)

Reviewer: ```suggestion
- Updated the keyboard interactions for the lookup component's dropdown menu. [#176](https://github.com/blackbaud/skyux-lookup/pull/176)
```

> - Fixed the lookup component to refocus the input box after closing the show more picker. [#193](https://github.com/blackbaud/skyux-lookup/pull/193)

Reviewer: ```suggestion
- Fixed the lookup component to focus on the input field after the show more picker closes. [#193](https://github.com/blackbaud/skyux-lookup/pull/193)
```
Same note as before that if we actually mean the input box component and not the lookup component's input field, we probably need different edits here.
